### DISCUSSION TITLE
GitHub copilot assumes language

### DISCUSSION THREAD

**Discussion by VideoCarp** [community/community#13324] [Copilot Conversations]
![image](https://user-images.githubusercontent.com/66365570/159677169-b561a2f2-37e4-470d-beaa-058d57da92f3.png)
GitHub copilot doesn't seem to check what programming language a file is in before making suggestions. Here, it's making what I'm guessing are C# suggestions for an F# program. Here's another example. 
![image](https://user-images.githubusercontent.com/66365570/159678225-1345f80e-4f0c-4177-b78e-85d105505f93.png)

**Comment by charlesroddie** [comment]
The file is not showing an `.fs` extension.
